satpy/etc/readers/fci_l1c_nc.yaml

```diff
@@ -6,109 +6,286 @@ reader:
+        - ir_97
+        - ir_105
+        - ir_123
+        - ir_133
   fci_l1c_hrfi:
     file_reader: !!python/name:satpy.readers.fci_l1c_nc.FCIL1cNCFileHandler
-    file_patterns: [ '{pflag}_{location_indicator},{data_designator},MTI{spacecraft_id:1d}+{data_source}-1C-RRAD-HRFI-{coverage}-{subsetting}-{component1}-BODY-{component3}-{purpose}-{format}_{oflag}_{originator}_{processing_time:%Y%m%d%H%M%S}_{facility_or_tool}_{environment}_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_{processing_mode}_{special_compression}_{disposition_mode}_{repeat_cycle_in_day:>04d}_{count_in_repeat_cycle:>04d}.nc' ]
+    file_patterns:
+      [
+        "{pflag}_{location_indicator},{data_designator},MTI{spacecraft_id:1d}+{data_source}-1C-RRAD-HRFI-{coverage}-{subsetting}-{component1}-BODY-{component3}-{purpose}-{format}_{oflag}_{originator}_{processing_time:%Y%m%d%H%M%S}_{facility_or_tool}_{environment}_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_{processing_mode}_{special_compression}_{disposition_mode}_{repeat_cycle_in_day:>04d}_{count_in_repeat_cycle:>04d}.nc",
+      ]
     expected_segments: 40
-    required_netcdf_variables:
-    - attr/platform
-    - data/{channel_name}/measured/start_position_row
-    - data/{channel_name}/measured/end_position_row
-    - data/{channel_name}/measured/radiance_to_bt_conversion_coefficient_wavenumber
-    - data/{channel_name}/measured/radiance_to_bt_conversion_coefficient_a
-    - data/{channel_name}/measured/radiance_to_bt_conversion_coefficient_b
-    - data/{channel_name}/measured/radiance_to_bt_conversion_constant_c1
-    - data/{channel_name}/measured/radiance_to_bt_conversion_constant_c2
-    - data/{channel_name}/measured/radiance_unit_conversion_coefficient
-    - data/{channel_name}/measured/channel_effective_solar_irradiance
-    - data/{channel_name}/measured/effective_radiance
-    - data/{channel_name}/measured/x
-    - data/{channel_name}/measured/y
-    - data/{channel_name}/measured/pixel_quality
-    - data/{channel_name}/measured/index_map
-    - data/mtg_geos_projection
-    - data/swath_direction
-    - data/swath_number
-    - index
-    - state/celestial/earth_sun_distance
-    - state/celestial/subsolar_latitude
-    - state/celestial/subsolar_longitude
-    - state/celestial/sun_satellite_distance
-    - state/platform/platform_altitude
-    - state/platform/subsatellite_latitude
-    - state/platform/subsatellite_longitude
-    - time
+    required_netcdf_variables: *required-variables
     variable_name_replacements:
       channel_name:
-      - vis_06_hr
-      - nir_22_hr
-      - ir_38_hr
-      - ir_105_hr
-
+        - vis_06_hr
+        - nir_22_hr
+        - ir_38_hr
+        - ir_105_hr
+  # Note: In The current file the 'MTI1-FCI-1C' which is a part of the file will be replaced by MTI1+FCI-1C, patterns have been added
+  # to maanage this
+  fci_l1c_af_vis_06:
+    file_reader: !!python/name:satpy.readers.fci_l1c_nc.FCIL1cNCFileHandler
+    file_patterns:
+      [
```

Comment:
Here also, 
```yaml
file_patterns:
  - first_pattern
  - second_pattern
```
would be more readable

Reply:
Ok I have done this modification, nice suggestion